src/components/document/sort-work-view.tsx

```diff
@@ -1,67 +1,179 @@
-import { observer } from "mobx-react";
-import { useStores } from "../../hooks/use-stores";
 import React, { useState } from "react";
+import { observer } from "mobx-react";
 import { SortWorkHeader } from "../navigation/sort-work-header";
+import { useStores, usePersistentUIStore, useAppConfig } from "../../hooks/use-stores";
 import { ICustomDropdownItem } from "../../clue/components/custom-select";
 import { DecoratedDocumentThumbnailItem } from "../thumbnail/decorated-document-thumbnail-item";
 import { DocumentModelType, getDocumentContext } from "../../models/document/document";
+import { NavTabModelType } from "../../models/view/nav-tabs";
 import { DocumentContextReact } from "./document-context";
 import { DEBUG_SORT_WORK } from "../../lib/debug";
 import { isSortableType } from "../../models/document/document-types";
+import { SortWorkDocumentArea } from "./sort-work-document-area";
 
 import "../thumbnail/document-type-collection.sass";
 import "./sort-work-view.scss";
 
-export const SortWorkView:React.FC = observer(function SortWorkView(){
+interface IProps {
+  tabSpec: NavTabModelType
+}
```

Comment:
I'd guess you don't really need this. The only thing I think you are using it for is the string "sort-work", which will always be the same for this component.